ruby/rails/security/brakeman/check-render-local-file-include.yaml

```diff
@@ -0,0 +1,30 @@
+rules:
+- id: check-render-local-file-include
+  mode: taint
+  pattern-sources:
+  - patterns:
+    - pattern: params[...]
+  pattern-sinks:
+  - patterns:
+    - pattern: |
+        render ...
+  message: Found request parameters in a call to `render`. This can allow end users
```

Comment:
What's a good recommended fix for this? Should I use a map of allowed tempaltes?

Reply:
updated